packages/components/src/site-thumbnail/index.tsx

```diff
@@ -0,0 +1,47 @@
+import classnames from 'classnames';
+import { ReactNode } from 'react';
+import './style.scss';
+import { useMshotsImg } from './use-mshots';
+import { getTextColorFromBackground } from './utils';
+
+const MSHOTS_OPTION = {
+	vpw: 1200,
+	vph: 1200,
+	w: 300,
+};
+
+type Props = {
+	backgroundColor?: string;
+	mShotsUrl?: string;
+	size?: 'small' | 'medium';
+	children?: ReactNode;
```

Comment:
What do you think about a `cacheKey` prop?

We could set it to a random value for a couple entries in `SiteThumbnailExample`, so that the loading effect is always triggered.

Reply:
I feel like something like cacheKey should be internal to the method and not exposed as a param. Is this only for testing purposes?

Reply:
> I feel like something like cacheKey should be internal to the method and not exposed as a param. Is this only for testing purposes?

@vykes-mac We'll also need some way for the Sites Dashboard to request a new screenshot for a site.

Reply:
> @vykes-mac We'll also need some way for the Sites Dashboard to request a new screenshot for a site.

I agree, was wondering if passing it as a param is the best approach or should we set it as an internal string in the `usemShotsImg` hook?

Reply:
@danielbachhuber  mshots has a `?requeue=true` param to regenerate thumbnails, This is what you are referencing interms of cache or the browser cache? If this flag is not set it will always server the previously generated thumbnail even when we clear the browser cache.